tekton/cronjobs/dogfooding/configmaps/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+resources:
+- labels-sync-hourly
+- prow-labels-sync-hourly
+- prow-config-hourly
+- prow-pluings-hourly
```

Comment:
should this be `prow-plugins-hourly`?

Reply:
Good catch, thanks. The name of the folder was also `prow-pluings-hourly` which is why I did not catch this when testing the build :)